src/RoomsSDKAdapter.js

```diff
@@ -36,6 +39,10 @@ export default class RoomsSDKAdapter extends RoomsAdapter {
 
     this.getRoomObservables = {};
     this.listenerCount = 0;
+
+    this.limit = 50;
+    this.roomObservables = {};
+    this.roomActivities = new Map();
```

Comment:
You have to keep in mind that the adapters are singleton objects, so they will be handling multiple rooms. Minor change but makes it clear that this is info from multiple rooms
```suggestion
    this.roomsActivities = new Map();
```

Reply:
You could think it like this. A `room` has activities. In order to get a `room` You must specify its `id` using the `get` method you are getting 1 room which has activities. Naming it `roomsActivities` makes it sounds like you want multiple rooms with activities.